_posts/2017-05-29_Widget

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "openSenseMap – jetzt gibt’s ein Widget"
+date: 2017-05-29
+author: Felix
+abstract: "Es gibt eine weitere Neuheit - ein Widget für die openSenseMap, mit dem die Daten auch außerhalb der Map angezeigt werden können"
+thumbnail: /images/blog_images/widget_screenshot.png
+lang: de
+---
+Daten aus der ganzen Welt
+============
+Unsere openSenseMap – eine Webplattform, mit der die Messdaten der senseBox direkt auf eine Karte übertragen werden. Egal ob in Deutschland, Brasilien oder Portugal: Du kannst dir einfach die Daten vor Ort herunterladen und analysieren. Doch jetzt gibt es eine Neuheit!
+
+Das neue Widget
+============
+Unser ehemaliger Praktikant Andreas Usdenski hat ein Widget für die Messdaten programmiert. Dieses kannst du in deine eigene Webseite einbinden, wodurch sich deine Werte auch außerhalb der openSenseMap anzeigen lassen. Das Widget sieht nicht nur cool aus, sondern macht auch den Umgang mit deinen Daten leichter.
+
+Und so geht’s:
+============
+Die Nutzung ist ganz einfach und beinhaltet nur zwei kurze Schritte:
+Zuerst wählt du auf der openSenseMap eine senseBox aus und kopierst den letzten Teil der Web-Adresse deines Browsers. Das ist deine senseBox ID.
+
+Um das Widget in deine Webseite zu integrieren, kopierst du folgenden HTML-Befehl in deine Webseite:
+<div data-sensebox-id="your-senseBox-Id" style="width:400px;height:600px;"></div> <script src="https://sensebox.de/opensensemap-widget/script.js"></script>
```

Comment:
Versuch mal diesen Teil in einen <Code> oder <pre> Tag einzupacken. @felixerdy Sind dafür Styles da?